api/app/services/dry_types_to_json.rb

```diff
@@ -0,0 +1,89 @@
+class DryTypesToJson
```

Comment:
I'd put this in a module inside of services. For example, `ApiDocumentation::DryTypesToJson`